src/components/form/FieldElement.tsx

```diff
@@ -16,18 +16,24 @@ interface FieldElementProps extends FormElementProps {
     endpoint?: string;
     add?: boolean;
     placeholder?: string;
+    searchPromptText?: string;
 }
 
 interface FieldState {
     field: SearchResult;
     errors: string[];
 }
 
-export const placeholder: string =
+export const PLACEHOLDER: string =
     'Enter the name of an existing group, or create a new group to add the contact to';
-export const notFound: string = 'Invalid field name';
+export const NOT_FOUND: string = 'Invalid field name';
 
 export default class FieldElement extends React.Component<FieldElementProps, FieldState> {
+    public static defaultProps = {
```

Comment:
took this opportunity to use the `defaultProps` property over custom logic in render